vendor.mod

```diff
@@ -28,6 +28,8 @@ require (
 	github.com/containerd/containerd v1.6.15
 	github.com/containerd/continuity v0.3.0
 	github.com/containerd/fifo v1.0.0
+	github.com/containerd/nydus-snapshotter v0.3.0-alpha.5
+	github.com/containerd/stargz-snapshotter v0.11.3
```

Comment:
Can we use the latest version

Reply:
The highest we can go is [v0.11.4](https://github.com/containerd/stargz-snapshotter/releases/tag/v0.11.4), this is the last version that contains `github.com/containerd/stargz-snapshotter/snapshot/overlayutils` used by the version of buildkit that we have, so we need to update buildkit in order to update the stargz snapshotter.

Reply:
@AkihiroSuda @rumpl I think v0.11.4 is fine now because this patch uses only `fs/source` pkg and there is no difference between v0.11.4 and the latest in that pkg. In the future, when we import other part of stargz-snapshotter code into moby, we might need to upgrade it to the latest.

Reply:
> The highest we can go is [v0.11.4](https://github.com/containerd/stargz-snapshotter/releases/tag/v0.11.4), this is the last version that contains github.com/containerd/stargz-snapshotter/snapshot/overlayutils used by the version of buildkit that we have, so we need to update buildkit in order to update the stargz snapshotter.

Related PR for 0.10 is in https://github.com/moby/buildkit/pull/3318. It's merged, but there's no new 0.10.x release yet

perhaps @tonistiigi knows if there's plans to do more 0.10 patch releases (for vendoring / go modules)